python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -1243,6 +1243,34 @@ def my_apply(df, unused):
     tm.assert_series_equal(result, expected)
 
 
+@pytest.mark.parametrize(
+    "op",
+    [
+        "__iadd__",
+        "__iand__",
+        "__ifloordiv__",
+        "__imod__",
```

Comment:
`__imatmul__` is implemented for `Series`, so please test that. This test only appears to cover `DataFrame`.

Reply:
Done 👍